x-pack/plugins/security_solution/public/common/components/exceptions/use_add_exception.tsx

```diff
@@ -111,6 +116,23 @@ export const useAddOrUpdateException = ({
           });
         }
 
+        if (shouldBulkClose === true) {
+          const filter = getQueryFilter(
+            '',
+            'kuery',
+            [],
+            ['.siem-signals'],
+            exceptionItemsToAddOrUpdate,
```

Comment:
The query for bulk closing has to account for the `event` fields in the original event being moved to `original_event` in the signal - https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.ts#L57

Reply:
Does this mean that i'll have to replace any exception item fields that start with `event.` with `signal.original_event.` before creating the ES Filter?

Reply:
Yeah in general `signal.original_event` holds the original event's `event` fieldset and `event.` in the signal refers to the "signal creation" event rather than the original event - so replacing all `event.` uses with `signal.original_event.` would be safest in case we use more signal specific `event` fields later on. Right now the only difference between I know of between `event` and `signal.original_event` is that `event.kind` is hardcoded to `signal`.